python/cac_tripplanner/cac_tripplanner/settings.py

```diff
@@ -0,0 +1,96 @@
+"""
+Django settings for cac_tripplanner project.
```

Comment:
I added code to load these types of settings from /etc/cac_secrets. I think this is cleaner than throwing the entire settings.py through the templating engine.